ompi/mca/osc/ucx/osc_ucx_active_target.c

```diff
@@ -260,29 +248,19 @@ int ompi_osc_ucx_post(struct ompi_group_t *group, int assert, struct ompi_win_t
             uint64_t curr_idx = 0, result = 0;
 
             /* do fop first to get an post index */
-            status = opal_common_ucx_atomic_fetch(ep, UCP_ATOMIC_FETCH_OP_FADD, 1,
-                                                  &result, sizeof(result),
-                                                  remote_addr, rkey, mca_osc_ucx_component.ucp_worker);
-            if (status != UCS_OK) {
-                opal_output_verbose(1, ompi_osc_base_framework.framework_output,
-                                    "%s:%d: ucp_atomic_fadd64 failed: %d\n",
-                                    __FILE__, __LINE__, status);
-            }
+            opal_common_ucx_atomic_fetch(ep, UCP_ATOMIC_FETCH_OP_FADD, 1,
```

Comment:
seems broken to not check the error code.. but it wasn't introduced with this PR

Reply:
@yosefe right, but assumed flush can't return error code (or it failed permanently which cause complete isconnect)